docs/api.rst

```diff
@@ -9356,14 +9356,51 @@ Querying general information
 
 .. http:get:: /api/(version)/info
 
-   Doing a GET on the info endpoint will return general information about rotki. Under the version key we get info on the version of rotki. If there is a newer version then ``"download_url"`` will be populated. If not then only ``"our_version"`` and ``"latest_version"`` will be. There is a possibility that latest version may not be populated due to github not being reachable. Also we return the data directory
+   Doing a GET on the info endpoint will return general information about rotki. Under the version key we get info on the current version of rotki. Also we return the data directory
```

Comment:
Why are you making two different endpoints? Just add all the information under 1 endpoint. There is no need to have multiple endpoints and force multiple queries with a roundtrip from frontend/user to backend to get this information.